config/space-presets.json

```diff
@@ -48,7 +48,7 @@
     }
   },
   {
-    "id": "5apps",
+    "id": "xmpp-5apps",
```

Comment:
But then it conflicts with the [computed `id`](https://github.com/67P/hyperchannel/pull/112/files#diff-be00797e644f8bf6fa99549fc3b20ffcR49) which is based on the name and is used for the URLs.